packages/block-library/src/table-of-contents/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import metadata from './block.json';
+import save from './save';
+import transforms from './transforms';
+
+const { name } = metadata;
+
+export { metadata, name };
+
+export const settings = {
+	title: __( 'Table of Contents' ),
+	description: __(
+		'Add a list of internal links allowing your readers to quickly navigate around.'
```

Comment:
I think this description needs to be updated since the block no longer automatically adds anchors to the headings.

Reply:
True. Maybe "Add a list of headers allowing your readers to quickly navigate through your post." instead, and somewhere else make it obvious to users how to make them links (maybe with popovers or similar)?

Reply:
I'd say "headings" rather than "headers". Also, the contributor guide recommends [avoiding describing features as "allowing" something](https://developer.wordpress.org/block-editor/contributors/document/copy-guide/#two-cut-phrases-that-inflate-your-word-count-without-actually-adding-meaning).

How's this?

"A list of headings to summarize your post. Add HTML anchors to Heading blocks to automatically link them here."